CHARTER.md

```diff
@@ -0,0 +1,61 @@
+# Charter for the Vega Projects
+
+## Scope and Goals
+
+This is the organizational charter for [Vega](https://github.com/vega/vega), [Vega-Lite](https://github.com/vega/vega-lite), [Vega-Altair](https://github.com/altair-viz/altair), [Vega-Embed](https://github.com/vega/vega-embed), [Vega-Datasets](https://github.com/vega/vega-datasets), [Vega-Themes](https://github.com/vega/vega-themes), [Vega-Tooltips](https://github.com/vega/vega-tooltip), [Vega-Editor](https://github.com/vega/editor), and [VL-Convert](https://github.com/vega/vl-convert). These projects are collectively referred to as the "Vega projects" in this document.
+
+The Vega projects are in the [vega](https://github.com/vega) and [altair-viz](https://github.com/altair-viz) GitHub organizations.
+
+## Mission
+
+The Vega projects provide an ecosystem of tools for declarative data visualizations. The various components of this ecosystem build on each other providing different levels of abstraction.
+
+## Steering Committee
+
+### Purpose
+
+The Steering Committee will be responsible for all technical oversight, project approval and oversight, policy oversight, and trademark management for the Organization.
```

Comment:
Typo? “Project approval and oversight” Should it be “project approval oversight”?

Reply:
I copied this from Altair. I think it means approving projects and overseeing projects. The steering committee is responsible for approving projects and not just overseeing the approval. Makes sense? We could shorten this to "project approval" since we have a separate point for "policy oversight". I'll change it for now.